tests/beakerlib/Tuned-takes-too-long-to-reload-start-when-ulimit/main.fmf

```diff
@@ -8,3 +8,5 @@ relevancy: |
     distro = rhel-4, rhel-5, rhel-6: False
 summary: Test for BZ#1663412 (TuneD takes too long to reload/start when \"ulimit)
 framework: beakerlib
+require:
+  - library(tuned/basic
```

Comment:
On first sight, typo missing closing parenthesis?

Reply:
Cut and pasta error. Fixed now.